### DISCUSSION TITLE
Q:

### DISCUSSION THREAD

**Discussion by sdqOS** [walinejs/waline#2028] [Q&A]
### Describe the bug | 问题描述

评论系统不知道为什么用不了

### Website URL | 问题网站

https://sdq3.link/valine.html

### Where your waline deploy? | 服务部署在哪里？

Vercel (Default)

### Where your comment data store? | 数据存储在哪里？

LeanCloud(https://leancloud.app)

---
### Describe the bug | Problem description

I don't know why the comment system doesn't work

### Website URL | Problem Website

https://sdq3.link/valine.html

### Where your waline deploy? | Where is the service deployed?

Vercel (Default)

### Where your comment data store? | Where is your comment data store?

LeanCloud(https://leancloud.app)

**Comment by lizheming** [comment]
https://comments.sdq.link serverURL 有问题，自行处理一下

---
There is a problem with https://comments.sdq.link serverURL, handle it yourself

**Reply by sdqOS** [reply]
是的有帖子链接吗？或者是教程

---
Yes is there a link to the post? Or a tutorial

**Reply by sdqOS** [reply]
现在还是有问题

---
There is still a problem
